### PR TITLE
Priya/bugfix

### PR DESCRIPTION
**Pull request by Svpriyaa2808**
Requesting review

### CODE REVIEW

File: html/changepage.html [hunk 1]
```diff
@@ -24,42 +24,57 @@ <h1 class="header__page-title">
...
+                            height="20" viewBox="0 0 20 20">
+                                <path
+                                    d="M16.4 4c0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4c0 0.961 0.568 1.784 1.384 2.167-0.082 1.584-1.27 2.122-3.335 2.896-0.87 0.327-1.829 0.689-2.649 1.234v-4.121c0.825-0.38 1.4-1.209 1.4-2.176 0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4c0 0.967 0.576 1.796 1.4 2.176v7.649c-0.825 0.379-1.4 1.208-1.4 2.175 0 1.326 1.074 2.4 2.4 2.4s2.4-1.075 2.4-2.4c0-0.961-0.568-1.784-1.384-2.167 0.082-1.583 1.271-2.122 3.335-2.896 2.030-0.762 4.541-1.711 4.64-4.756 0.829-0.378 1.409-1.21 1.409-2.181zM6 2.615c0.764 0 1.384 0.619 1.384 1.385 0 0.764-0.62 1.385-1.384 1.385s-1.384-0.621-1.384-1.385c0-0.766 0.619-1.385 1.384-1.385zM6 17.385c-0.766 0-1.385-0.621-1.385-1.385 0-0.766 0.619-1.385 1.385-1.385 0.764 0 1.384 0.619 1.384 1.385 0 0.764-0.62 1.385-1.384 1.385zM14 5.385c-0.766 0-1.385-0.621-1.385-1.385 0-0.766 0.619-1.385 1.385-1.385 0.764 0 1.384 0.619 1.384 1.385 0 0.764-0.62 1.385-1.384 1.385z">
+                                </path>
                             </svg>
                             <span>Branching</span>
```
Comment (VaishaliGovindaraj) [inline html/changepage.html]: Change to Branches

Comment (Svpriyaa2808) [reply html/changepage.html]: sure

**Comment by TobiasKjernell** [comment]
in your Changespage: Change your head to "Changes" on row 19 in your html.

File: styles/changepage.css [hunk 2]
```diff
@@ -32,30 +32,40 @@
...
+    height: 400px;
    justify-content: center;
- 
 }
 
 .command {
     border-radius: 10px;
     margin: 8px;
     width: 200px;
+    height: 300px;
```
Comment (ranjithavelusamy) [inline styles/changepage.css:64]: its good

**Review by ranjithavelusamy — APPROVED** [review]

**Review by VaishaliGovindaraj — APPROVED** [review]

File: html/changepage.html [hunk 3]
```diff
@@ -24,42 +24,57 @@ <h1 class="header__page-title">
...
+                                </path>
                             </svg>
                             <span>Changes</span>
                         </a>
                     </li>
                     <li class="side-nav__item">
-                        <a href="#" class="side-nav__link">
-                            <svg class="side-nav__icon">
-                                <use xlink:href="img/sprite.svg#icon-map"></use>
+                        <a href="../html/branch.html" class="side-nav__link">
```
Comment (andrewkalumba) [inline html/changepage.html]: you can change the ../html/branch.html to just ./branch because its in the same folder with your branch

Comment (andrewkalumba) [reply html/changepage.html]: you can do this to all the branches except main/index.html

**Review by andrewkalumba — APPROVED** [review]
looks great

**Review by TobiasKjernell — APPROVED** [review]

**Pull request merged**